source/Meadow.Core/Update/UpdateService.cs

```diff
@@ -572,6 +580,49 @@ private void DisplayTree(DirectoryInfo di)
         }
     }
 
+    // In order to not require the use of a library, this was taken from
+    // RFC7515, Appendix C: https://datatracker.ietf.org/doc/html/rfc7515#appendix-C
+    private string base64UrlDecode(string arg)
```

Comment:
There are libraries that support parsing and validating JWTs like [Microsoft.IdentityModel.JsonWebTokens](https://learn.microsoft.com/en-us/dotnet/api/microsoft.identitymodel.jsonwebtokens?view=msal-web-dotnet-latest), but this seemed like a simpler solution that didn't require additional libraries.